Libplanet/Net/Swarm.MessageHandlers.cs

```diff
@@ -212,25 +210,28 @@ private void ProcessTxIds(TxIds message)
                 message.Ids.Select(txid => txid.ToString())
             );
 
-            IStagePolicy<T> stagePolicy = BlockChain.StagePolicy;
-            ImmutableHashSet<TxId> newTxIds = message.Ids
-                .Where(id => !_demandTxIds.ContainsKey(id))
-                .Where(id => !stagePolicy.Ignores(BlockChain, id))
-                .ToImmutableHashSet();
+            HashSet<TxId> required = GetRequiredTxIds(message.Ids);
```

Comment:
And, it seems that we need a new test case for basic features. ( Codecov is complaining... 😅 )